### PR TITLE
feat/palette-enablement

### PR DESCRIPTION
**Pull request by KenjiCrosland**
## Description

...

### Design:
- [ ] Reviewed with designer and meets expectations

### Cross-browser testing:
- [ ] Chrome
- [ ] Firefox
- [ ] Edge
- [ ] Safari
- [ ] iOS
- [ ] Android

### Unit testing:
- [ ] Sufficient unit test coverage (see unit test best practices for ideas)
- [ ] Snapshot updates are explained with comment and reference the relevant source code change

### A11y:
- [ ] Meets WCAG AA standards

### Documentation:
- [ ] API docs created/updated
- [ ] Examples created/updated

### CODE REVIEW

File: postcss.config.js [hunk 1]
```diff
@@ -1,8 +1,12 @@
 const process = require('process');
+const path = require('path');
 
 module.exports = {
   syntax: 'postcss-scss',
   plugins: {
+    'postcss-jit-props': {
+      files: [path.resolve('node_modules/@rei/cdr-tokens/dist/rei-dot-com/css/cdr-tokens.css')],
```
Comment (benjag) [inline postcss.config.js]: What is this doing? Streamlining the final build output or just there for dev purposes?

File: src/components/rating/CdrRating.vue [hunk 2]
```diff
@@ -132,24 +139,23 @@ export default defineComponent({
...
-        :class="mapClasses(style, 'cdr-rating__icon', 'cdr-rating__100')"
+        :size="size"
         :key="`rating-whole-${star}`"
         aria-hidden="true"
       />
-      <span
+      <component
         v-if="remainder !== '00'"
-        :class="mapClasses(style, 'cdr-rating__icon', `cdr-rating__${remainder}`)"
+        :is="'CdrStar' + remainder"
```
Comment (benjag) [inline src/components/rating/CdrRating.vue]: Let's stick to string interpolation for consistency

:is="\`CdrStar${remainder}\`"

Comment (benjag) [reply src/components/rating/CdrRating.vue]: No action needed though, I'll take care of this when merging the features

File: postcss.config.js (hunk 1 above)
Comment (KenjiCrosland) [reply postcss.config.js]: It's a "just-in-time" plugin which only important css properties being used in the kitchen sink build. This is kind of a POC for the plugin. It doesn't need to be there as it only affects the kitchen sink

File: src/components/rating/CdrRating.vue (hunk 2 above)
Comment (KenjiCrosland) [reply src/components/rating/CdrRating.vue]: Sounds good

File: postcss.config.js (hunk 1 above)
Comment (benjag) [reply postcss.config.js]: Ok, I'm going to remove as to not add an extra dependency (plus I want to remove as much postcss as I can/deprecate the `extract-css` from the build in a future update)

**Pull request merged**